src/shared/config/apiUrls.ts

```diff
@@ -230,7 +231,31 @@ export const getAPIQueryParams = ({
   // note: could this be moved to useNSQuery?
   if (facetField === undefined) {
     facetField = defaultFacets.get(namespace);
+
+    // Exceptional case for showing proteome facet in UniProtKB search if the query is about an organism
+    if (namespace === Namespace.uniprotkb && facetField) {
+      const proteomeFacet = FacetsEnum.Proteome;
+      if (
+        query.includes('organism_id') &&
+        !facetField.includes(proteomeFacet)
+      ) {
+        const index =
+          facetField.findIndex((facet) => facet === FacetsEnum.ModelOrganism) +
+            1 || facetField.length;
+
+        facetField.splice(index, 0, proteomeFacet);
+      } else if (
+        !query.includes('organism_id') &&
+        facetField.includes(proteomeFacet)
+      ) {
+        const index = facetField.findIndex(
+          (facet) => facet === FacetsEnum.Proteome
+        );
+        facetField.splice(index, 1);
+      }
+    }
   }
```

Comment:
Was this done to match the behaviour on legacy?
If I understood that bit of code well it adds the proteomes facet on the side if one searches for an organism, right?
Now that I'm playing with it I don't think we need to match that, because the issue is that it will show multiple proteomes if there are more than 1, and it won't be clear which one is the best one to choose.
With your work on the suggestion sentence at the top I think it's enough